ansible/roles/kubernetes/defaults/main.yml

```diff
@@ -4,6 +4,6 @@ kubernetes_cni_version: "{{ kubernetes.cni_version if kubernetes_cni_version_def
 kubernetes_enable_cached_images: "{{ kubernetes.enable_cached_images if kubernetes_enable_cached_images_defined else False }}"
 kubernetes_cached_images: "{{ kubernetes.cached_images if kubernetes_cached_images_defined else [] }}"
 kubernetes_apt_key_url: "{{ kubernetes.apt_key_url if kubernetes_apt_key_url_defined else 'https://packages.cloud.google.com/apt/doc/apt-key.gpg' }}"
-kubernetes_apt_repo_string: "{{ kubernetes.apt_repo_string if kubernetes_apt_repo_string_defined else 'deb http://apt.kubernetes.io/ kubernetes-' ~ ansible_distribution_release | lower ~ ' main' }}"
+kubernetes_apt_repo_string: "{{ kubernetes.apt_repo_string if kubernetes_apt_repo_string_defined else 'deb http://apt.kubernetes.io/ kubernetes-xenial main' }}"
```

Comment:
maybe worth a comment that we have to hardcode this because k/release only release xenial artifacts?

Reply:
👍 Agree